test/categories/exceptions/reraised.pls:5:5: ERROR:
Exception: PANIC! AAAAA
    Caused by 'Panic'
Call trace:
    test/categories/exceptions/reraised.pls:9:9
    test/categories/exceptions/reraised.pls:14:5
Reraised at:
    test/categories/exceptions/reraised.pls:10:28
    test/categories/exceptions/reraised.pls:15:24
  | 
5 |     raise (Panic("AAAAA"))
  |     ^^^^^^^^^^^^^^^^^^^^^^